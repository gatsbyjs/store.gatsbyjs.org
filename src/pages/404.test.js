import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './404';

describe('Pages: 404', () => {
  it('renders a 404 page', () => {
    const component = shallow(<NotFound />);

    expect(component).toMatchSnapshot();
  });
});
