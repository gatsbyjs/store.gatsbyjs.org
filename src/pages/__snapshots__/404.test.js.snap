// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages: 404 renders a 404 page 1`] = `
<Styled(div)>
  <Styled(h1)>
    Whoops - That Page Doesn’t Exist (404)
  </Styled(h1)>
  <Styled(p)>
    Looks like the page you requested either doesn’t exist or has been moved. If you think this is an error or ended up at this page by following a link, please
     
    <a
      href="https://github.com/gatsbyjs/gatsby/issues/new"
    >
      open an issue
    </a>
     
    to let us know.
  </Styled(p)>
</Styled(div)>
`;
